content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +115,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up VM machines
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Install the Debian package with the Envoy sidecar:
+<!-- TODO: find a better way to reference release in the URL instead of manually sync... -->
+
+1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM. For example
 
     {{< text bash >}}
-    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
+    $ export GCE_NAME="your-gce-instance"
+    $ gcloud compute scp {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
```

Comment:
This may not work since users may have a wrong project and zone set up.

Reply:
If there's concern about this, maybe we could add something like this: See [this link](https://cloud.google.com/sdk/gcloud/reference/compute/scp) for additional parameters to point `gcloud compute scp` to different GCP projects or zones.

Could add that above, before "For example".

Reply:
I just added `--project` and `--zone` argument since we already referenced them as env var.